services/api/src/modules/packs/packs.service.ts

```diff
@@ -737,6 +739,59 @@ export default class PacksService {
     return true
   }
 
+  async untransferredPacks({
+    externalId,
+    locale = DEFAULT_LOCALE,
+  }: LocaleAndExternalId) {
+    const packs = await PackModel.query()
+      .withGraphJoined('[collectibles, owner]')
+      .where('owner.externalId', externalId)
+      .whereNull('collectibles.ownerId')
+      .whereNotNull('collectibles.address')
```

Comment:
Exclude unminted collectibles, for now.

Reply:
Theoretically, the minting will have occurred in most cases by this point in most cases. But in this particular scenario, the user will be re-prompted again once unminted assets they've claimed are finally minted?

Reply:
A user shouldn't end up in a situation where this is hit and finds unminted packs. It happened to me locally as I was working on this feature where the minting had failed. Can chat a bit more about how we wanna handle this after standup today.

Reply:
Pretty much. There are two cases the unminted ones could be found with this query (and as a result need to be excluded):

1. This endpoint is hit just after they've purchased a pack, but before the minting has completed. For this, excluding them makes sense.
2. If you purchase a pack but we fail to start the minting process or some error occurs that makes it impossible to finish the minting process. Then those packs' minting processes will need to be retried. Today, we have no mechanism to handle that and I don't think it should be part of this.

So the question is, what do we do in the latter case?

Reply:
Ah ok gotcha. Hmm good question. I can see having a task that looks for completed payments without minted assets, but wondering how we'd avoid minting collectibles when the process had been started but was not complete. 🤔